[19 09] brave 1.4.95 to 1.5.112

JeffLabonte opened the pull request:
Update brave from the version 1.4.95 to 1.4.96.

(cherry picked from commit 3f6d356654db6641895d274b1fdac8a15c452c06)
Reason: Keep the browser to the latest stable version

Backport the PR #81788 and #82894

<!-- Nixpkgs has a lot of new incoming Pull Requests, but not enough people to review this constant stream. Even if you aren't a committer, we would appreciate reviews of other PRs, especially simple ones like package updates. Just testing the relevant package/service and leaving a comment saying what you tested, how you tested it and whether it worked would be great. List of open PRs: <https://github.com/NixOS/nixpkgs/pulls>, for more about reviewing contributions: <https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#chap-reviewing-contributions>. Reviewing isn't mandatory, but it would help out a lot and reduce the average time-to-merge for all of us. Thanks a lot if you do! -->
###### Motivation for this change

Keep the browser up-to-date for everyone

###### Things done

Update the version and the checksum

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

JeffLabonte committed:
brave: 1.4.95 -> 1.4.96

Update brave from the version 1.4.95 to 1.4.96.

(cherry picked from commit 3f6d356654db6641895d274b1fdac8a15c452c06)
Reason: Keep the browser to the latest stable version

rht reviewed (approved):
LGTM

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/pr-approved-browser-brave-backport/6122/1

JeffLabonte committed:
brave: 1.4.96 -> 1.5.112

keep brave up-to-date

(cherry picked from commit 418e3e41cf22753911c5f474d8bd89252ec76ce0)
Reason: Browsers should be kept up-to-date for security reasons
